src/ai-bundle/src/Command/ChatCommand.php

```diff
@@ -126,29 +115,29 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $agentArg = $input->getArgument('agent');
-        $this->agentName = \is_string($agentArg) ? $agentArg : '';
+        $agentName = \is_string($agentArg) ? $agentArg : '';
 
         // In non-interactive mode, agent is required
-        if (!$this->agentName && !$input->isInteractive()) {
+        if (!$agentName && !$input->isInteractive()) {
```

Comment:
this check can be removed entirely. The argument is always required by the time `execute` is called, but it is validated by Symfony.